Mage.Tests/src/test/java/org/mage/test/multiplayer/BlatantThieveryTest.java

```diff
@@ -24,7 +25,7 @@
 
     @Override
     protected Game createNewGameAndPlayers() throws GameException, FileNotFoundException {
-        Game game = new FreeForAll(MultiplayerAttackOption.MULTIPLE, RangeOfInfluence.ALL, 0, 20);
+        Game game = new FreeForAll(MultiplayerAttackOption.MULTIPLE, RangeOfInfluence.ALL, new VancouverMulligan(0), 20);
```

Comment:
Default mulligan constructor can be replaced with standard call like `MulliganType.GAME_DEFAULT.getMulligan(0)` or same call (if wizards try to change default mulligan then no needs to change it all around).